ldclient/__init__.py

```diff
@@ -11,6 +13,27 @@
                 "firstName", "lastName", "avatar", "name", "anonymous"]
 
 
+"""Settings."""
+client = None
+api_key = None
+start_wait = 5
+config = Config()
+
+_lock = threading.Lock()
+
+
+def get():
+    try:
+        _lock.acquire()
```

Comment:
@pkaeding suggested looking at ways to not always acquire the lock as it could be heavily contended during normal use.